src/app/services/chart.service.ts

```diff
@@ -79,9 +79,13 @@ export class ChartService {
 
     loadChartData(): void {
         let queries = [];
-
         let options = this.dataQueryOptions;
 
+        // send an empty notification to indicate that data is currently loading
+        if (this.chartDataObserver) {
+            this.chartDataObserver.next({});
```

Comment:
Looks like next should be passed an `[]` instead of `{}` to satisfy the type checker